flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -542,14 +542,14 @@ private void connect(Integer headOfChain, StreamEdge edge) {
 
 		physicalEdgesInOrder.add(edge);
 
-		Integer downStreamvertexID = edge.getTargetId();
+		Integer downstreamVertexID = edge.getTargetId();
```

Comment:
nit: `downstream` -> `downStream` & `upstream` -> `upStream`. I quickly git grepped the code, and `downstream` is only used in text/docs. `downStream` is used in the code.